editor/editor_inspector.cpp

```diff
@@ -3131,8 +3131,38 @@ void EditorInspector::update_tree() {
 	bool disable_favorite = false;
 	VBoxContainer *category_vbox = nullptr;
 
+	HashMap<String, String> &reorder_from_to = get_root_inspector()->reorder_prop_from_to;
+	HashSet<String> &reorder_to_set = get_root_inspector()->reorder_prop_to_set;
+	reorder_from_to.clear();
+	reorder_to_set.clear();
+
+	get_root_inspector()->updating_tree = true;
 	List<PropertyInfo> plist;
 	object->get_property_list(&plist, true);
+	get_root_inspector()->updating_tree = false;
+
+	if (!reorder_from_to.is_empty()) {
+		HashMap<String, List<PropertyInfo>::Element *> move_elements;
+		for (List<PropertyInfo>::Element *pe = plist.front(); pe; pe = pe->next()) {
+			PropertyInfo &p = pe->get();
+			if (reorder_from_to.has(p.name)) {
+				move_elements.insert(p.name, pe);
+			}
+			if (reorder_to_set.has(p.name)) {
+				move_elements.insert(p.name, pe);
+			}
+		}
+
+		for (KeyValue<String, String> kv : reorder_from_to) {
+			List<PropertyInfo>::Element *k_element = move_elements.get(kv.key);
+			List<PropertyInfo>::Element *v_element = move_elements.get(kv.value);
+			plist.move_before(k_element, v_element);
```

Comment:
```suggestion
			ERR_CONTINUE(k_element == nullptr);
			ERR_CONTINUE(v_element == nullptr);
			plist.move_before(k_element, v_element);
```
Try this, if it solves it (and you get a warning) you have an issue with the fetch here

Reply:
Was not this part then

Reply:
Hmm, looks like the github test still fails, but it works fine locally

I feel like I'm not using the preprocessor correctly or something

Reply:
Try building with the same flags as the test, but with debug symbols so you can get a crash trace

Reply:
ok I'll try